.travis.yml

```diff
@@ -69,6 +69,15 @@ jobs:
       script:
           - make $MAKEJOBS
           - make $MAKEJOBS bench
+    - stage: Benchmark
+      os: linux
+      compiler: gcc
+      before_script:
+          - export COVERAGE=OFF
+          - export CFLAGS="-O3"
+      script:
+          - make $MAKEJOBS
```

Comment:
is this line redundant and the same other compilers/oses benchmarks?

Reply:
No, because we need to redefine parent's  "script" here